networkx/classes/ordered.py

```diff
@@ -0,0 +1,55 @@
+"""
+OrderedDict variants of the default base classes.
+
+These classes are especially useful for doctests and unit tests.
```

Comment:
One thing is for sure: My debatable comment should not have been in the source code for that module. I'll gut it soon.